services/provider/proto/provider.proto

```diff
@@ -118,13 +118,30 @@ message StorageConfigResponse{
     SystemAttributes systemAttributes = 3;
 }
 
+message RbdDriverRequirements {
+  bool generateOmapInfo = 1;
+}
+
+message CephFsDriverRequirements {
+}
+
+message NfsDriverRequirements {
+}
+
+message CsiDrivers {
+  RbdDriverRequirements DesiredRbdDriver = 1;
```

Comment:
Let make sure we are naming the field in accordance to our terminology. As we decided to call these "requirements" I would argue the name of the field should be `RbdDriverRequirments`